jetstream/cli.py

```diff
@@ -202,6 +204,12 @@ def rerun(project_id, dataset_id, experiment_slug, config_file):
 
         config = spec.resolve(experiment)
 
+        # delete all tables previously created when this experiment was analysed
+        client = BigQueryClient(project_id, dataset_id)
+        normalized_slug = bq_normalize_name(experiment.normandy_slug)
+        table_name_re = f"(statistics_)?{normalized_slug}.*"
```

Comment:
nit: can we make this more restrictive? slugs will be freeform in the future so it's likely that one slug will be a substring of another